css/properties/position.json

```diff
@@ -16,7 +16,10 @@
             },
             "firefox": {
               "version_added": "1",
-              "notes": "Before Firefox 57, absolute positioning did not work correctly when applied to elements inside tables that have <a href='https://developer.mozilla.org/docs/Web/CSS/border-collapse'><code>border-collapse</code></a> applied to them (<a href='https://bugzil.la/1379306'>bug 1379306</a>)."
+              "notes": [
+                "Before Firefox 57, absolute positioning did not work correctly when applied to elements inside tables that have <a href='https://developer.mozilla.org/docs/Web/CSS/border-collapse'><code>border-collapse</code></a> applied to them (<a href='https://bugzil.la/1379306'>bug 1379306</a>).",
+                "Before Firefox 30, absolute positioning of table rows and row groups was not supported (<a href='https://bugzil.la/63895'>bug 63895</a>)."
+              ]
             },
             "firefox_android": {
               "version_added": "4"
```

Comment:
For completeness, maybe we ought to copy the notes from desktop to Android?

Reply:
Good point, done!